File: CommandEventExample/PHP/MyApp/Payloads/Foo.php
<?php

declare(strict_types=1);

namespace MyApp\Payloads;

use MyLibrary\IPayload;

final class Foo implements IPayload
{
    private string $Value;

    public function __construct(string $value)
    {
        $this->Value = $value;
    }

    public function Value() : string
    {
        return $this->Value;
    }
}

File: Java/CommandsEvents/src/MyApp/Payloads/Foo.java
package MyApp.Payloads;

import MyLibrary.IPayload;

public final class Foo implements IPayload
{
    private final String Value;

    public Foo(String value) {
        this.Value = value;
    }

    public String Value()
    {
        return this.Value;
    }
}
